semgrep_output_v1.atd

```diff
@@ -1282,7 +1291,11 @@ type finding = {
   end_column: int;
 
   message: string;
-  (* int|string until minimum version exceeds 1.32.0, then string *)
+  (* int|string until minimum version exceeds 1.32.0, then int?
```

Comment:
Does minimum version exceed 1.32.0 yet? How far away from that are we?

Reply:
Not sure what it is ... Jonas was managing that and I don't know who took over this.

Reply:
Ok I found it in the code. It's 1.31.0 and was bumped to that 3 months ago. It feels like it should be safe to bump this above 1.32.0 in that case? I don't think anyone "owns" this right now sadly but I can dig a bit.